### ISSUE TITLE
Add /helpop from Essentials as a channel

### ISSUE THREAD

**Issue by SDpyro** [DiscordSRV/DiscordSRV#970]
It would be very helpful if you could add support for the /helpop command from Essentials
Exactly how dynmap has its channel.

Thanks!

**Comment by Bbro122** [comment]
Just saying it's possible to do this with a slight modification in the alerts.yml section of the plugin
`  - Trigger: /helpop
    Channel: helpop
    Conditions:
      - '#player.hasPermission("essentials.helpop") == true'
    Embed:
      Color: "#00ff00"
      Author:
        ImageUrl: "{embedavatarurl}"
        Name: "{name} >> ${#allArgs}"`

**Comment by HappyDev345** [comment]
> Just saying it's possible to do this with a slight modification in the alerts.yml section of the plugin
> 
> * Trigger: /helpop
>   Channel: helpop
>   Conditions:
>   
>   * '#player.hasPermission("essentials.helpop") == true'
>     Embed:
>     Color: "#00ff00"
>     Author:
>     ImageUrl: "{embedavatarurl}"
>     Name: "{name} >> ${#allArgs}"

so this works if i copy and paste?

**Comment by Dinty1** [comment]
You’d have to set the channel to either the channel id or define a “helpop” one in the Channels section of config.yml

**Comment by HappyDev345** [comment]
how? do you just put 1182988210842894406 for example in the channel line?

**Comment by Dinty1** [comment]
Yes. If you need more help please open a thread in #alerts in the discord server

**Comment by Dinty1** [comment]
Use #alerts in the discord server.
